go/tricorder/metric_test.go

```diff
@@ -1737,6 +1775,16 @@ func TestListNoChangeSubType(t *testing.T) {
 	t.Error("panic expected")
 }
 
+func TestNoAssignedUnitNoAdd(t *testing.T) {
+	bucketer := NewGeometricBucketer(1, 1000)
+	dist := newDistribution(bucketer, false)
+	defer func() {
+		recover()
```

Comment:
Thx. Just sent you the PR. I ended up following my already existing convention of panic* variables for panic messages.